h2o-algos/src/main/java/hex/gam/MatrixFrameUtils/GamUtils.java

```diff
@@ -93,6 +92,12 @@ public static GLMParameters copyGAMParams2GLMParams(GAMParameters parms, Frame t
     setParamField(parms, glmParam, true, field2);
     glmParam._train = trainData._key;
     glmParam._valid = valid==null?null:valid._key;
+    glmParam._nfolds = 0; // always set nfolds to 0 to disable cv in GLM.  It is done in GAM
+    glmParam._fold_assignment = Model.Parameters.FoldAssignmentScheme.AUTO;
+    glmParam._keep_cross_validation_fold_assignment = false;
```

Comment:
Can be removed. Default is the same.

Reply:
@pavel: In this case, I am copying GAM parameters to GLM parameters. If I want cross-validation and set _keep_cross_validation_fold_assignment, and therefore after the copying of parameters, GLM parameter _keep_cross_validation_fold_assignment will also be set.  Hence, I need to manually set them.